app/src/main/res/drawable/splash_image.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+  <item android:drawable="@color/color_def_oppia_green" />
```

Comment:
I believe a static check will fail becaus we only use component colors in xml for darkmode support. Please see: https://github.com/oppia/oppia-android/wiki/Dark-Mode